File: PythonLearning/python_q_nd_a/prime.py
for n in range (2,10):
    for x in range(2,n):
        if n % x == 0:
            print "%s is composite number" %(n)
            break
    else:
        print "%s is prime number" %(n)


  

def fib(n):

    a,b = 0,1
    while a<n:
        print a
        a,b = b,a+b


fib(1000) 

File: perl_preparation/prime.pl
#!usr/bin/perl -w
use strict;

sub prime {
    my $number = shift;
    my $d = 2;
    my $sqrt = sqrt $number;
    while(1) {
        if ($number%$d == 0) {
            return 0;
        }
        if ($d < $sqrt) {
            $d++;
        } else {
            return 1;
        }
    }
}

my $number = $ARGV[0];
print prime($number);
